tests/network/conftest.py

```diff
@@ -217,3 +221,18 @@ def cnao_deployment(hco_namespace):
         namespace_name=hco_namespace.name,
         deployment_name=CLUSTER_NETWORK_ADDONS_OPERATOR,
     )
+
+
+@pytest.fixture(scope="session", autouse=True)
+def network_sanity(hosts_common_available_ports, junitxml_plugin):
+    """
+    Perform verification that the cluster is a multi-nic one otherwise exit run
+    """
+    LOGGER.info("Verify cluster running network tests is a multi-nic one")
+    if len(hosts_common_available_ports) <= 1:
+        exit_pytest_execution(
```

Comment:
Can you please explain why use this one instead of a simple `assert hosts_common_available_ports > 1` ?
This one also seems to collect must-gather and do other stuff.

Reply:
For consistency. It can be expanded for other checks. Must-gather collection would be skipped for this one.